hpOneView/resources/networking/fcnetworks.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we rename the module to fc_networks.py? I believe fc_networks improves readability.